preview-maker.js

```diff
@@ -170,16 +161,11 @@ const makePrev = async (themeBuffer, themeName, themeAuthor, template) => {
     }
     const chooseHsl = rgbToHsl(color);
     let hueDifference = getDifference(chooseHsl, windowBackgroundWhite, `hue`);
-    let saturationDifference = getDifference(
-      chooseHsl,
-      windowBackgroundWhite,
-      `saturation`
-    );
-    let lightnessDifference = getDifference(
-      chooseHsl,
-      windowBackgroundWhite,
-      `lightness`
-    );
+    let hueDifference = Math.abs(chooseHsl.hue - rgbToHsl(windowBackgroundWhite).hue);
```

Comment:
You've declared the same variable just one line above. this code will fail even to parse.